### ISSUE TITLE
Provide per-layer learning rate policies?

### ISSUE THREAD

**Issue by erogol** [BVLC/caffe#3071]
I believe tuning your net is a crux of present state of art deep learning. Then, setting different lr_policy options per layer is a inevitable requirement for this. I believe introducing this to Caffe would be useful.

**Comment by shelhamer** [comment]
Closing as overcomplication: layer-wise learning rate scaling offers hyperparameters enough.
